src/reactviews/common/utils.ts

```diff
@@ -29,21 +25,21 @@ export function formatString(str: string, ...args: any[]): string {
 
 /**
  * Gets the theme type based on the theme passed in
- * @param theme the theme of the react webview
+ * @param themeKind the theme of the react webview
```

Comment:
Can you update the comments on this (and related) utility methods that state what they're mapping from/to?  e.g. "maps from VS Code theme type to Fluent theme type", with a link to the docs page that says what the valid fluent types are.

Reply:
This doesn't do any mapping now. It is just exposing the vscode theme kind.